controllers/pipeline/pipeline_adapter.go

```diff
@@ -452,6 +452,7 @@ func (a *Adapter) getAllPipelineRunsForSnapshot(snapshot *applicationapiv1alpha1
 // prepareSnapshot prepares the Snapshot for a given application and the updated component (if any).
 // In case the Snapshot can't be created, an error will be returned.
 func (a *Adapter) prepareSnapshot(application *applicationapiv1alpha1.Application, component *applicationapiv1alpha1.Component, newContainerImage string, newComponentSource *applicationapiv1alpha1.ComponentSource) (*applicationapiv1alpha1.Snapshot, error) {
+	logger := a.logger.WithApp(*application)
```

Comment:
https://github.com/redhat-appstudio/integration-service/blob/e86f164b6828d7d29bfd87378f1f8ff74ed83463/controllers/pipeline/pipeline_controller.go#L115

application has been already added to logger, why is this needed here?

Reply:
Thanks finding this. When I check the logs in my unittest, I found the application is not logged, so I add that line. You remind me, the controller called `.WithApp`, to see the log in my unittest, I should add that line to pipeline_adapter_test.go https://github.com/redhat-appstudio/integration-service/blob/ae921ce4e43b231117e29477f7f3e1164ea0b9b7/controllers/pipeline/pipeline_adapter_test.go#L138